### ISSUE TITLE
remove lims from points() lists after passing through the set/extract

### ISSUE THREAD

**Issue by jordansread** [USGS-R/gsplot#279]
``` r
gs <- gsplot() %>% 
    points(y=1, x=2, xlim=c(0,3),ylim=c(0,3), col="blue", pch=18)

gs$view$window$ylim
[1] 0 3
gs$view$points$ylim
[1] 0 3
```

I know why this is the case, but we end up with ylim=c(0,3) both in the gs$view$points and gs$view$window if the user specifies it. 

The reason this happens (I think, if I remember correctly) is because we want to keep around the user-specified args so they don't get overwritten by the next addition of `points` or something else. 

I would like to see all those types of args end up in window, because the things in `points` should really just be those that are part of the call to graphics::points in the rendering stage.

perhaps window needs `user.ylim` and `data.ylim` to handle this type of thing.

**Comment by jordansread** [comment]
Fixed w/ #290

**Comment by jordansread** [comment]
``` r
gs <- gsplot() %>% 
    points(y=1, x=2, xlim=c(0,3),ylim=c(0,3), col="blue", pch=18)
gs$view.1.2$points
$x
[1] 2

$y
[1] 1

$col
[1] "blue"

$pch
[1] 18

```
